ProcessMaker/processmaker
# FOUR-13016 | Remove Background Color and Border from Slide Task

# Issue
Ticket: [FOUR-13016](https://processmaker.atlassian.net/browse/FOUR-13016)

Slide tasks appear in the modal with a background color and border. The tasks should have the same background color as the modal, and no border to separate the task content from the rest of the modal content.

# Previous UI
![Screen Shot 2023-12-21 at 11 46 45 AM](https://github.com/ProcessMaker/processmaker/assets/73713665/ea64bf70-c0fb-433d-8f01-dd1f7bba6369)

# Updated UI
![UpdateModalSlide](https://github.com/ProcessMaker/processmaker/assets/73713665/a0227d68-1c98-4f59-a12c-f48309f3cde1)

# How to Test
1. Go to branch `observation/FOUR-13016` in `processmaker`.
2. Go to Processes → Guided Templates.
3. Select the Invoice Approval template.
4. Click the 'Use Now' on the first slide.
5. The information in the second slide should not have a border around it and should not have a background color that does not match the modal's color.

ci:next

# Code Review Checklist


[FOUR-13016]: https://processmaker.atlassian.net/browse/FOUR-13016?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
